charts/alfresco-connector-hxi/templates/secret-repository.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if not .Values.repository.existingSecret.name -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  {{- $ctx := dict "Values" (dict "nameOverride" (printf "%s-%s" ($.Values.nameOverride | default $.Chart.Name) "repository")) "Chart" $.Chart "Release" $.Release }}
+  name: {{ template "alfresco-connector-hxi.fullname" $ctx }}
+  labels:
+    {{- include "alfresco-connector-hxi.labels" . | nindent 4 }}
+data:
+  REPOSITORY_USERNAME: {{ .Values.repository.username | default "" | b64enc | quote }}
+  REPOSITORY_PASSWORD: {{ .Values.repository.password | default "" | b64enc | quote }}
+  REPOSITORY_CLIENT_ID: {{ .Values.repository.clientId | default "" | b64enc | quote }}
+  REPOSITORY_CLIENT_SECRET: {{ .Values.repository.clientSecret | default "" | b64enc | quote }}
```

Comment:
can we drop all of this if now we are using the version override stuff?

Reply:
There are two ways to obtain repo version, so it depends if we want to preserve the possibility to have an option? 🤔 Or we are ok with forcing admin to manually provide the version?

Reply:
Even more these properties will be required by `prediction-applier`